interfaces/seccomp/backend_test.go

```diff
@@ -71,25 +80,33 @@ func (s *backendSuite) TestName(c *C) {
 func (s *backendSuite) TestInstallingSnapWritesProfiles(c *C) {
 	s.InstallSnap(c, interfaces.ConfinementOptions{}, ifacetest.SambaYamlV1, 0)
 	profile := filepath.Join(dirs.SnapSeccompDir, "snap.samba.smbd")
-	// file called "snap.sambda.smbd" was created
-	_, err := os.Stat(profile)
+	// file called "snap.sambda.smbd.in" was created
+	_, err := os.Stat(profile + ".in")
 	c.Check(err, IsNil)
+	// and got compiled
+	c.Check(s.snapSeccomp.Calls(), DeepEquals, [][]string{
+		{"snap-seccomp", "compile", profile + ".in", profile + ".bpf"},
+	})
```

Comment:
The use of ".in" here doesn't seem to match up with backend.go which doesn't reference ".in" that I can see. What's happening here?